pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -31,6 +31,15 @@
 _slice_msg = "slice indices must be integers or None or have an __index__ method"
 
 
+def test_unsure_about_the_name_and_location():
```

Comment:
we need to make a test_iloc.py file in this directory, might as well do it in this PR